Float.TinCan.LocalLRSServer/LRSServer.cs

```diff
@@ -42,11 +42,6 @@ public LRSServer(string address = "http://127.0.0.1", ushort port = 8080, ILRSSe
         /// </summary>
         public event EventHandler<StatementEventArgs> StatementReceived;
 
-        /// <summary>
-        /// Event to be raised when the local LRS receives agent profile document information.
-        /// </summary>
-        public event EventHandler<AgentProfileDocumentEventArgs> AgentProfileDocumentReceived;
```

Comment:
@bbeversdorf IMHO we should keep the event to avoid a breaking change to the API.

Reply:
Keep it and never call it?
SparkLearn is the only code base that is really calling this and it is being updated

Reply:
I would still raise the event in response to a PUT/POST request the way the framework does now.

I think we'll make our lives easier if we treat the framework as its own codebase and respect semantic versioning and only make breaking changes when needed.